### DISCUSSION TITLE
TheHive - markdown images not working as expected

### DISCUSSION THREAD

**Discussion by glwallum** [Security-Onion-Solutions/securityonion#6156] [Unsupported Versions] [answered]
Hello

I am working with TheHive to integrate images into the markdown/description fields of the cases and alerts. 

- If I use an external image URL it will load correctly and show the image.
- If I use an internal image URL the image will not load. 
- I have verified thehive node can do a wget to the image, and that the image loads on my machine accessing thehive url.

I have tried hosting the internal image on IIS and python simple-http-server. Both will not load.

Any advice would be appreciated

**Comment by dougburks** [comment, marked as answer]
As mentioned at https://blog.securityonion.net/2020/12/security-onion-2-in-2020-and-2021.html, we have plans to change how we do case management. This became even more important when [TheHive announced their licensing change](https://medium.com/strangebee-announcements/thehive-turns-5-and-adopts-a-model-shaped-for-the-future-95f908719c31). Based on that, you might not want to put too much time and effort into customizing your TheHive installation.
